task1.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+let base = 1000000;
+
+function addition(arr1, arr2) {
+  if (arr1.length < arr2.length) {
+    let tmp = arr1;
+    arr1 = arr2;
+    arr2 = tmp;
+  }
+  if (arr2[0] === 0 && arr2.length === 1) return arr1;
+  let len1 = arr1.length;
+  let len2 = arr2.length;
+  while (arr2.length !== len1) {
+    arr2.push(0);
```

Comment:
It's better to avoid unnecessary modifications of the input array.

Reply:
Why these modifications are unnecessary?

Reply:
Because there is a way to solve the problem without them